### PR TITLE
[WIP] Result

### LINKED ISSUE #29: Result
https://ant.design/components/result/

### PR DESCRIPTION
**Pull request by vertonghenb**
Fixes #29

### CODE REVIEW

File: docs/Append.AntDesign.Documentation/wwwroot/css/documentation-styles.min.css [hunk 1]
```diff
@@ -1 +1 @@
...eline.ant-timeline-alternate .ant-timeline-item-head,.ant-timeline.ant-timeline-right .ant-timeline-item-head,.ant-timeline.ant-timeline-label .ant-timeline-item-head{margin-left:-4px;}.ant-timeline.ant-timeline-alternate .ant-timeline-item-head-custom,.ant-timeline.ant-timeline-right .ant-timeline-item-head-custom,.ant-timeline.ant-timeline-label .ant-timeline-item-head-custom{margin-left:1px;}.ant-timeline.ant-timeline-alternate .ant-timeline-item-left .ant-timeline-item-content,.ant-timeline.ant-timeline-right .ant-timeline-item-left .ant-timeline-item-content,.ant-timeline.ant-timeline-label .ant-timeline-item-left .ant-timeline-item-content{left:calc(50% - 4px);width:calc(50% - 14px);text-align:left;}.ant-timeline.ant-timeline-alternate .ant-timeline-item-right .ant-timeline-item-content,.ant-timeline.ant-timeline-right .ant-timeline-item-right .ant-timeline-item-content,.ant-timeline.ant-timeline-label .ant-timeline-item-right .ant-timeline-item-content{width:calc(50% - 12px);margin:0;text-align:right;}.ant-timeline.ant-timeline-right .ant-timeline-item-right .ant-timeline-item-tail,.ant-timeline.ant-timeline-right .ant-timeline-item-right .ant-timeline-item-head,.ant-timelin
```
Comment (JorisCl) [inline docs/Append.AntDesign.Documentation/wwwroot/css/documentation-styles.min.css:1]: .... **.ant-alert.ant-alert-no-icon**{padding:8px 15px;}**.ant-alert.ant-alert-no-icon** .ant-alert-close-icon{top:12.0005px;}.ant-alert.ant-alert-closable{padd...


it is correct that ".ant-alert.ant-alert-no-icon" is present twice in the styles file, once with a padding and once without any style. (see snippit above)

File: source/Append.AntDesign/wwwroot/css/ant-design-blazor.min.css [hunk 2]
```diff
@@ -1 +1 @@
...
...ght:58.33333333%;}.ant-col-sm-offset-14{margin-left:58.33333333%;}.ant-col-sm-order-14{order:14;}.ant-col-sm-13{display:block;flex:0 0 54.16666667%;max-width:54.16666667%;}.ant-col-sm-push-13{left:54.16666667%;}.ant-col-sm-pull-13{right:54.16666667%;}.ant-col-sm-offset-13{margin-left:54.16666667%;}.ant-col-sm-order-13{order:13;}.ant-col-sm-12{display:block;flex:0 0 50%;max-width:50%;}.ant-col-sm-push-12{left:50%;}.ant-col-sm-pull-12{right:50%;}.ant-col-sm-offset-12{margin-left:50%;}.ant-col-sm-order-12{order:12;}.ant-col-sm-11{display:block;flex:0 0 45.83333333%;max-width:45.83333333%;}.ant-col-sm-push-11{left:45.83333333%;}.ant-col-sm-pull-11{right:45.83333333%;}.ant-col-sm-offset-11{margin-left:45.83333333%;}.ant-col-sm-order-11{order:11;}.ant-col-sm-10{display:block;flex:0 0 41.66666667%;max-width:41.66666667%;}.ant-col-sm-push-10{left:41.66666667%;}.ant-col-sm-pull-10{right:41.66666667%;}.ant-col-sm-offset-10{margin-left:41.66666667%;}.ant-col-sm-order-10{order:10;}.ant-col-sm-9{display:block;flex:0 0 37.5%;max-width:37.5%;}.ant-col-sm-push-9{left:37.5%;}.ant-col-sm-pull-9{right:37.5%;}.ant-col-sm-offset-9{margin-left:37.5%;}.ant-col-sm-order-9{order:9;}.ant-col-sm-8{display:bl
```
Comment (JorisCl) [inline source/Append.AntDesign/wwwroot/css/ant-design-blazor.min.css:1]: .ant-alert.ant-alert-no-icon{padding:8px 15px;}.ant-alert.ant-alert-no-icon .ant-alert-close-icon{top:12.0005px;}.

Is it correct that ".ant-alert.ant-alert-no-icon" is present twice in this style file, once with padding and once without?

File: docs/Append.AntDesign.Documentation/wwwroot/css/documentation-styles.min.css (hunk 1 above)
Comment (vertonghenb) [reply docs/Append.AntDesign.Documentation/wwwroot/css/documentation-styles.min.css]: Good catch, I'll check it out. 
However in other PR's we should discard the changes to the .css files (WebCompiler Bug on Ubuntu)

File: source/Append.AntDesign/wwwroot/css/ant-design-blazor.min.css (hunk 2 above)
Comment (vertonghenb) [reply source/Append.AntDesign/wwwroot/css/ant-design-blazor.min.css]: Good catch, I'll check it out. 
However in other PR's we should discard the changes to the .css files (WebCompiler Bug on Ubuntu)

**Pull request merged**